docker/database.sh

```diff
@@ -23,3 +23,8 @@ echo '--- RUNNING POST INSTALL'
 
 echo '--- COPY CLEAN DATABASE TO BE USED IF NO VOLUMES ARE MOUNTED'
 cp -R /var/lib/postgresql/12/main/ ./fresh_db
+
+# We end up with extraneous file folders in the repo root, so delete them now
+rm -r files
+rm -r localisation
+rm -r preferences
```

Comment:
Deleting unnecessary folders from the Docker container makes it less confusing to figure out which folders are being used for what.